lib/presentation/view/auth/widget/auth_form_widget.dart

```diff
@@ -1,27 +1,55 @@
+import 'package:asset_tracker/core/config/constants/string_constant.dart';
+import 'package:asset_tracker/core/config/theme/extension/app_size_extension.dart';
+import 'package:asset_tracker/core/config/theme/extension/responsive_extension.dart';
+import 'package:asset_tracker/core/widgets/custom_padding.dart';
 import 'package:flutter/material.dart';
 
 class AuthFormWidget extends StatelessWidget {
   const AuthFormWidget({
     super.key,
     required this.label,
     required this.isObs,
-    required this.size,
     required this.formController,
     required this.validaor,
   });
 
-  final Size size;
+
   final String label;
   final TextEditingController? formController;
   final FormFieldValidator<String>? validaor;
   final bool isObs;
 
+  ///* first param: TextEditing Controller
+  ///* second param: FormFieldValidator<String>? validator
+  factory AuthFormWidget.email(TextEditingController? emailController,
+      FormFieldValidator<String>? emailValiator) {
+    return AuthFormWidget(
```

Comment:
Aşağıdaki ifadeler belli olduğu için direkt constructor da tanımlayabilirsin. Bu da bir yöntem ama constructor tanımlayabiliyorken o şekilde yazarsan daha hoş görüntü oluşur. en azından benim açımdan 😄 

```
AuthFormWidget.password({
/// TextEditingController parameter for password input
TextEditingController? passController,
      ///Pasword validatior function
      FormFieldValidator<String>? passValidator,
      }):this(
      label: DefaultLocalStrings.passwordText,
      isObs: true,
      formController: passController,
      validaor: passValidator,
      );
```